pipeline_steps/common.groovy

```diff
@@ -78,13 +76,13 @@ def void install_ansible(){
       SHA=\$(git rev-parse HEAD)
     popd
 
-    curl -s "\${REPO_BASE}/index" > index ||:
```

Comment:
Do we still need to create/update the index in `rpc_jobs/build_gating_venv.yml` then?